### ISSUE TITLE
Allow specifying the release when publishing sources

### ISSUE THREAD

**Issue by loomchild** [nuxt-community/sentry-module#73]
### What problem does this feature solve?

Currently `release` config option is only used when reporting errors, but is ignored when publishRelease is active. The specified release should be taken into account when uploading sources instead of always relying on commit hash.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/sentry-module/issues/c54">#c54</a>)</em></sub></div>

**Comment by aldarund** [comment]
Right now you can specify it via webpackConfig.release param.
In v3 it will default to config.release but still could be overwriten by webpackConfig.release

**Comment by loomchild** [comment]
Great, thanks a lot. Feel free to close this ticket if you don't need it anymore.

**Comment by napnapnapnap** [comment]
Hi, any guidance on how to do it ? I tried overwriting the webpack release like this : 
```
    build: {
        plugins: [
            new webpack.LoaderOptionsPlugin({
                options: {
                    config: {
                        release: "2.3.10"
                    }
                }
            })
        ],
    }
```
but it doesn't work. Thanks for any help.

**Comment by rchl** [comment]
@napnapnapnap no, aldarund meant to specify it in sentry's module config.

```
webpackConfig: {
  release: '2.2',
},
```

**Comment by napnapnapnap** [comment]
@rchl 
Thanks, works perfectly.

**Comment by aldarund** [comment]
closed in #72
default to `config.release` value

**Comment by loomchild** [comment]
Thanks a lot, I confirm it works as expected in v3.0.0!
